scripts/us_census/acs5yr/subject_tables/common/base.py

```diff
@@ -0,0 +1,87 @@
+"""Base data loader for ACS subject tables"""
+
+import os
+import csv
+import json
+import pandas as pd
+
+from io import BytesIO
+from requests import get
+from zipfile import ZipFile
+
+import generateColMap
+
+
+_TEMPLATE_TMCF = """Node: E:SubjectTable->E{index}
+typeOf: dcs:StatVarObservation
+variableMeasured: C:SubjectTable->dcs:{statVar}
+measurementMethod: dcs:CesnsusACS{estimatePeriod}yrSurvey
+observationDate: C:SubjectTable->ObservationDate
+observationAbout: C:SubjectTable->ObservationAbout
+value: C:SubjectTable->{statVar}
+scalingFactor: C:SubjectTable->{scalingFactor}
+unit: C:SubjectTable->{unit}
+"""
+
+_UNIT_TEMPLATE = """unit: dcs{unit}"""
+
+_IGNORED_VALUES = set(['-', '*', '***', '*****', 'N', '(X)', 'null'])
+
+class SubjectTableDataLoaderBase:
+  """Base dataloader specification for processing subjectTables"""
+  def __init__(self, tableID='S2702', acsEstimatePeriod='5', hasPercentages=True, config_json_path=None, outputPath=None):
```

Comment:
This can be hardcoded for now.

Drop tableID from default maybe.

Reply:
Alright :) I have made this change